PopPUNK/__main__.py

```diff
@@ -433,7 +464,8 @@ def main():
                                  queryList,
                                  assignments,
                                  model.within_label,
-                                 weights=weights)
+                                 weights = weights,
+                                 use_gpu = args.gpu_graph)
```

Comment:
~Agree that this is passed enough times that it would make more sense to set the model objects up with a GPU option (and device ID) which is set in the same way as threads~
See comment below